contrib/bear/template.py

```diff
@@ -0,0 +1,28 @@
+pkgname = "bear"
+pkgver = "3.1.4"
+pkgrel = 0
+build_style = "cmake"
+configure_args = [
+    "-DENABLE_UNIT_TESTS=ON",
+    "-DENABLE_FUNC_TESTS=OFF",  # need lit https://pypi.org/project/lit
+]
+hostmakedepends = [
+    "cmake",
+    "ninja",
+    "pkgconf"
+]
+makedepends = [
+    "fmt-devel",
+    "grpc-devel",
+    "nlohmann-json",
+    "spdlog-devel",
+]
+checkdepends = [
+    "gtest-devel",
+]
+pkgdesc = "Tool that generates a compilation database for clang tooling"
```

Comment:
```suggestion
pkgdesc = "Tool that generates a compilation database"
```

the json can be consumed by anything technically